repo/commit.go

```diff
@@ -18,7 +18,7 @@ func ForceCommit() *exec.Cmd {
 	cmd := exec.Command(
 		"git",
 		"commit",
-		"-m 'Commit dirty state'")
+		"-m 'Commit dirty state [ci skip]'") // skip the CI build since this commit was triggered by the build system, not by a user
```

Comment:
I personally prefer it at the beginning of the commit message, and `[skip ci]` sounds nicer to me.